### ISSUE TITLE
contracting agreements

### ISSUE THREAD

**Issue by jbenet** [papertrail/legal-docs#5]
Love that you've made these docs available. Do you have any "open-source friendly" contracting agreements? We find all the stock ones to be over-protective -- with all the work being open source there's less need for many of the standard clauses.

**Comment by troy** [comment]
Thanks :) What you see is what exists. These aren't specific to any particular license for work and I'm not sure whether it applies. If the employer elects to make work public, it's by definition no longer confidential, and nearly all NDAs are scoped to exactly that -- whatever is confidential.

Usual disclaimer applies: this is not legal advice. Consult an employment attorney for that :)

**Comment by jbenet** [comment]
thanks for the response @troy 

> If the employer elects to make work public, it's by definition no longer confidential, and nearly all NDAs are scoped to exactly that -- whatever is confidential.

Yeah, this makes sense and what i've discussed with lawyers before.

For the curious, i'm specifically looking for a simpler version of this: https://www.docracy.com/5572/consulting-agreement optimized for open source (without the need for the CIIAA and some other parts). 

Thanks anyway! will post back here if i find anything

**Comment by troy** [comment]
What do you expect to change because it's OSS? There's still IP to be assigned, even if the employer chooses to license it some other way. If you don't want the IP to be assigned at all, I don't think that's customary (OSS or not). Even unpaid contributor agreements often do that so that one entity can make decisions on behalf of all contributors.

**Comment by jbenet** [comment]
Yeah there are already CLA-s in place for this so the thought is that it's unnecessary to do it twice. There's an assymetry were as a contributor (not a contractor) you get a nicer path. Actually just spoke with our lawyers about this and we may be preparing a doc. Will update here if there are good results.
